config/config.go

```diff
@@ -11,7 +11,7 @@ import (
 var ErrInvalidTransport = fmt.Errorf("invalid transport")
 
 type GlobalConfig struct {
-	//TODO add support for global config as needed
+	CachePath string `yaml:"cache_path"`
```

Comment:
K8s uses camelCase, docker-compose uses snake_case. I guess docker-compose is more common to average dev. 👍

Reply:
excellent! I'm more of a snake_case fan :smile: